src/app/components/Loading.tsx

```diff
@@ -0,0 +1,15 @@
+import Backdrop from '@mui/material/Backdrop';
+import CircularProgress from '@mui/material/CircularProgress';
+import { useStepsContext, STEPS_NAMES } from '../context/stepsContext';
+
+export function Loading({ isLoading }: any) {
+    const value = useStepsContext()
+
+    return (
+        <>
+            <Backdrop color={"#20365F"} open={isLoading ? true : false} >
```

Comment:
might make sense to define a theme for your MUI components rather than defining a color manually here